mongodbatlas/resource_mongodbatlas_org_invitation_test.go

```diff
@@ -0,0 +1,196 @@
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMongoDBAtlasOrgInvitationConfig(orgID, name, initialRole),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckMongoDBAtlasOrgInvitationExists(resourceName, &invitation, t),
+					testAccCheckMongoDBAtlasOrgInvitationUsernameAttribute(&invitation, name),
+					testAccCheckMongoDBAtlasOrgInvitationRoleAttribute(&invitation, initialRole),
+					resource.TestCheckResourceAttrSet(resourceName, "org_id"),
+					resource.TestCheckResourceAttrSet(resourceName, "username"),
+					resource.TestCheckResourceAttrSet(resourceName, "invitation_id"),
+					resource.TestCheckResourceAttr(resourceName, "username", name),
+					resource.TestCheckResourceAttr(resourceName, "roles.#", "1"),
+				),
+			},
+			{
+				Config: testAccMongoDBAtlasOrgInvitationConfig(orgID, name, updateRoles),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckMongoDBAtlasOrgInvitationExists(resourceName, &invitation, t),
+					testAccCheckMongoDBAtlasOrgInvitationUsernameAttribute(&invitation, name),
+					testAccCheckMongoDBAtlasOrgInvitationRoleAttribute(&invitation, updateRoles),
+					resource.TestCheckResourceAttrSet(resourceName, "username"),
+					resource.TestCheckResourceAttrSet(resourceName, "invitation_id"),
+					resource.TestCheckResourceAttr(resourceName, "name", name),
+					resource.TestCheckResourceAttr(resourceName, "roles.#", "2"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccResourceMongoDBAtlasOrgInvitation_importBasic(t *testing.T) {
+	var (
+		resourceName = "mongodbatlas_invitations.test"
+		orgID        = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		name         = fmt.Sprintf("test-acc-%s", acctest.RandString(10))
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckMongoDBAtlasOrgInvitationDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMongoDBAtlasOrgInvitationConfig(orgID, name, []string{"mongodbatlas.testing@gmail.com"}),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(resourceName, "org_id"),
+					resource.TestCheckResourceAttrSet(resourceName, "name"),
+					resource.TestCheckResourceAttrSet(resourceName, "usernames.#"),
+					resource.TestCheckResourceAttrSet(resourceName, "invitation_id"),
+
+					resource.TestCheckResourceAttr(resourceName, "org_id", orgID),
+					resource.TestCheckResourceAttr(resourceName, "name", name),
+					resource.TestCheckResourceAttr(resourceName, "usernames.#", "1"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportStateIdFunc: testAccCheckMongoDBAtlasOrgInvitationStateIDFunc(resourceName),
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccCheckMongoDBAtlasOrgInvitationExists(resourceName string, invitation *matlas.Invitation, t *testing.T) resource.TestCheckFunc {
```

Comment:
[style] it's convention for `t` to always be first 
```suggestion
func testAccCheckMongoDBAtlasOrgInvitationExists(t *testing.T, resourceName string, invitation *matlas.Invitation) resource.TestCheckFunc {
```
Same the other change you did for this

Reply:
Thanks for this advice